tools/msp_sr_readmap_and_size_histograms/readmap.xml

```diff
@@ -1,239 +1,109 @@
-<tool id="Readmap" name="Generate readmap and histograms from alignment files" version="1.1.5">
-  <description>from sRbowtie aligment</description>
-  <requirements>
-        <requirement type="package" version="0.12.7">bowtie</requirement>
-        <requirement type="package" version="0.7.7">pysam</requirement>
-        <requirement type="package" version="3.1.2">R</requirement>
-        <requirement type="package" version="2.14">biocbasics</requirement>
-        <requirement type="package" version="1.9">numpy</requirement>
-  </requirements>
-<command interpreter="python">
-        readmap.py 
-	          #if $refGenomeSource.genomeSource == "history":
-         	    --reference_fasta  ## sys.argv[2]
-                    $refGenomeSource.ownFile ## index source
-          	  #else:
-                    #silent reference= filter( lambda x: str( x[0] ) == str( $refGenomeSource.series[0].input.dbkey ), $__app__.tool_data_tables[ 'bowtie_indexes' ].get_fields() )[0][-1]
-		    --reference_bowtie_index
-                    $reference
-          	  #end if
-		  --rcode
-		  $plotCode
-		  --output_readmap
-		  $readmap_dataframe
-		  --output_size_distribution
-		  $size_distribution_dataframe
-		  --minquery
-		  $minquery
-		  --maxquery
-		  $maxquery
-		  --input
-		  #for $i in $refGenomeSource.series
-    		    $i.input 
-		  #end for
-		  --ext
-		  #for $i in $refGenomeSource.series
-    		    $i.input.ext 
-		  #end for
-		  --label
-		  #for $i in $refGenomeSource.series
-    		    "$i.input.name" 
-		  #end for
-		  --normalization_factor
-		  #for $i in $refGenomeSource.series
-    		    $i.norm
-		  #end for
-		  #if $gff:
-		    --gff
-                    $gff
-                  #end if
-
+<tool id="Readmap" name="Generate readmap and histograms from alignment files" version="1.2.0">
```

Comment:
Yes, but after my rallye for upgrading Oases in metavisitor 1.2, I will not do it again before a while. If you bump msp_sr_readmap_and_size_histograms, which would be nice, I would like you to ensure tool/workflow/docker update process ;-)